src/data/finsupp/ne_locus.lean

```diff
@@ -0,0 +1,113 @@
+/-
+Copyright (c) 2022 Damiano Testa. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Damiano Testa
+-/
+import data.finsupp.basic
+
+/-!
+#  `ne_locus` for finitely supported functions
```

Comment:
```suggestion
# `{x | f x ≠ g x}` for finitely supported functions
```
or
```suggestion
# Locus of unequal values of finitely supported functions
```
or some other description that doesn't describe the object using itself!

Reply:
I prefer the second version, to avoid symbols in titles.